src/service/video-manager-impl.js

```diff
@@ -307,8 +397,18 @@ class VideoEntry {
    * @private
    */
   videoLoaded_() {
-    this.updateVisibility();
     this.loaded_ = true;
+
+    // Get the internal element (the actual video/iframe)
+    this.internalElement_ = scopedQuerySelector(
+        this.video.element,
+        'video, iframe'
+    );
+    // Get its style and classlist
+    this.internalStyle_ = this.internalElement_.style;
```

Comment:
I don't think we need to cache these. unlike `computedStyle`, `style` and `classList` should not cause any relayouts upon read.